apps/dashboard/src/contract-ui/tabs/claim-conditions/page.tsx

```diff
@@ -35,6 +37,25 @@ export const ContractClaimConditionsPage: React.FC<
     return <div>Loading...</div>;
   }
 
+  const hasNewClaimConditions = _detectFeatures(contractQuery.contract, [
+    // erc721
+    "ERC721ClaimConditionsV2",
+    "ERC721ClaimPhasesV2",
+    // erc1155
+    "ERC1155ClaimConditionsV2",
+    "ERC1155ClaimPhasesV2",
+    // erc20
+    "ERC20ClaimConditionsV2",
+    "ERC20ClaimPhasesV2",
+  ]);
+
+  const contractInfo = useMemo(() => {
```

Comment:
this should yell at you because you are creating a hook after an if statement, that's not allowed (not sure why lint is not blowing up?)